### PR TITLE
fix(env-tpl): fix env variable format

### PR DESCRIPTION
**Pull request by luabida**

### CODE REVIEW

File: .env.tpl [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 AIRFLOW__DATABASE__SQL_ALCHEMY_CONN=${AIRFLOW__DATABASE__SQL_ALCHEMY_CONN}
 AIRFLOW__WEBSERVER__SECRET_KEY=${AIRFLOW__WEBSERVER__SECRET_KEY}
-AIRFLOW__SMTP__SMTP_HOST=smtp.sendgrid.net
-AIRFLOW__SMTP__SMTP_STARTTLS=False
-AIRFLOW__SMTP__SMTP_SSL=False
-AIRFLOW__SMTP__SMTP_USER=apikey
-AIRFLOW__SMTP__SMTP_PASSWORD=<generated-api-key>
-AIRFLOW__SMTP__SMTP_PORT=587
-AIRFLOW__SMTP__SMTP_MAIL_FROM=<your-from-email>
+AIRFLOW__SMTP__SMTP_HOST=${AIRFLOW__SMTP__SMTP_HOST:-"smtp.sendgrid.net"}
```
Comment (xmnlab) [inline .env.tpl:14]: Hey @luabida , could you remove the defaul value from here. I think we don't need to suggest any provider here.

**Review by xmnlab — CHANGES_REQUESTED** [review]
Thanks for working on that!
Just added a comment for a small issue.
Other than that, it lgtm! Thanks!!

File: .env.tpl (hunk 1 above)
Comment (luabida) [reply .env.tpl]: Done!

**Review by xmnlab — APPROVED** [review]
Lgtm. Thanks!

**Pull request merged**